### PR TITLE
Countupdatetest

### PR DESCRIPTION
**Pull request by shawnkumar**
just adds a test to check that select count(*) is returning the right count

### CODE REVIEW

File: upgrade_through_versions_test.py [hunk 1]
```diff
@@ -372,6 +375,23 @@ def _check_counters(self):
...
+        cursor.execute("use upgrade;")
+
+        expected_num_rows = len(self.row_values)
+
+        countquery = SimpleStatement("SELECT COUNT(*) FROM cf;", consistency_level=consistency_level)
+        result = cursor.execute(countquery)
+
+        if result is not None:
+            actual_num_rows = result[0]
+            assert actual_num_rows == expected_num_rows, "SELECT COUNT(*) returned %s when expecting %s" % (actual_num_rows,expected_num_rows)
```
Comment (EnigmaCurry) [inline upgrade_through_versions_test.py]: Let's use self.assertEquals here, it provides much better reporting in the case of a failure.

**Comment by EnigmaCurry** [comment]
+1

**Comment by ptnapoleon** [comment]
Can you squash before merge?

**Comment by ptnapoleon** [comment]
Beautiful. +1 from me as well.

**Pull request merged**